sharktank_models/quality_tests/README.md

```diff
@@ -0,0 +1,39 @@
+## Regression tests
+
+### Adding your own model
+
+- To add your own model, create a directory under `quality_tests` and add JSON files that correspond to the submodels. Please follow the [JSON file schema in this README file](#required-and-optional-fields-for-the-json-model-file)
+
+### How to run
+
+- Example command to run quality tests for a specific model
+
+```
+python sharktank_models/quality_tests/ \
+    -rpFe \
+    --log-cli-level=info \
+    --timeout=600 \
+    --durations=0
+```
+
+### Required and optional fields for the JSON model file
+
+| Field Name                          | Required | Type    | Description                                                                                                                                      |
+| ----------------------------------- | -------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------ |
+| inputs                              | optional | array   | An array of objects that provides the input blob and the expected input value (ex: `{"source" :"", "value": ""}`, the value field is optional)   |
+| outputs                             | optional | array   | An array of objects that provides the output blob and the expected output value (ex: `{"source" :"", "value": ""}`, the value field is optional) |
+| real_weights                        | optional | string  | URL that provides the real weights blob                                                                                                          |
+| mlir                                | required | string  | URL that provides the MLIR blob                                                                                                                  |
+| pipeline_mlir                       | optional | string  | URL that provides the MLIR blob                                                                                                                  |
+| device                              | optional | string  | The device to run the threshold tests on                                                                                                                  |
```

Comment:
nit: looks like the final `|` characters in this table are not aligned. You could omit them or add the right number of spaces

![image](https://github.com/user-attachments/assets/62404faa-78fb-4af6-8d6b-349ae9a6dd13)